### DISCUSSION TITLE
Hi Mrbt0907, got questions or want to learn aspects of weather?

### DISCUSSION THREAD

**Discussion by NitroRiced** [CosmicHikari/Weather2-Remastered#45] [General]
This is not really a suggestion, i just want to try help you out if you aren't quite sure about some aspects of weather or want to learn how weather systems work, just to save some time for you researching about weather. I am pretty knowledgeful about weather, so if you aren't sure, please go ahead and ask me a question :)

Also is it possible for me to be able to test the latest alpha of the mod?

**Comment by Mrbt0907** [comment]
I'll have some questions for you next week when I am active in development again. I hope I can learn some new things to better the update on launch.

**Comment by NitroRiced** [comment]
Sure Thing, ask me whenever

**Comment by Mrbt0907** [comment]
Moved this for the future update.
